cdec/installation-manager-core/src/main/java/com/codenvy/im/utils/HttpTransport.java

```diff
@@ -148,15 +150,29 @@ public Path download(String path, Path destinationDir) throws IOException {
         return download(path, "GET", MediaType.APPLICATION_OCTET_STREAM, destinationDir);
     }
 
+    public HttpTransport noProxy() {
+        noProxy = true;
```

Comment:
Looks like there can be issues with concurent access. We can avoid this by adding parameter no proxy or by making HttpTransport as no singleton and creating httpTransportFactory